Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:2.7
```

Comment:
Another side note: can we deploy this somewhere to test out everything, like a free AWS server or something?